recipe/meta.yaml

```diff
@@ -30,8 +30,11 @@ requirements:
     - zlib
 
 test:
+  requires:
+    - make
```

Comment:
the linter failed here, I think.  These requirements must be in the `run` section above, if at all